purchase_requisition_incoterm/models/purchase_order.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    incoterm_id = fields.Many2one(
+        "account.incoterms",
+        compute="_compute_incoterm_id",
+        store=True,
+    )
+
+    @api.depends("requisition_id")
+    def _compute_incoterm_id(self):
+        for order in self:
+            if order.requisition_id and order.requisition_id.incoterm_id:
+                order.incoterm_id = order.requisition_id.incoterm_id
+            elif not order.requisition_id:
+                order.incoterm_id = False
+
+    @api.model
+    def create(self, vals):
```

Comment:
Is this methodl still needed? I guess not.

Reply:
The module seems to work without it, so I removed it